src/averageAges.js

```diff
@@ -15,11 +15,32 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let totalAge = 0;
+  let fitManCounter = 0;
+
+  people.forEach(person => {
```

Comment:
try to use the array methods like `map`, `reduce`, `filter` etc.

Reply:
> try to use the array methods like `map`, `reduce`, `filter` etc.

remade but am i wrong with fact that this makes perfomance worse?